app/controllers/admin_applications_controller.rb

```diff
@@ -0,0 +1,18 @@
+class AdminApplicationsController < ApplicationController
```

Comment:
For this project, this controller organization is great. IT is possible, however, that this controller could get VERY big if all the admin paths lead here. So another option is to use an admin *directory* in your controller directory, and then create 2 different controllers under the admin directory -> "/admin/shelters_controller.rb" and "admin/applications_controller.rb". In each of those controller files, the class declaration would look like this: `class Admin::ApplicationsController < ApplicationController` with the `Admin::` namespace to indicate that this controller lives within the admin directory.

We'll talk more about this concept on Tuesday in class!